comps/dataprep/src/integrations/arangodb.py

```diff
@@ -430,11 +431,35 @@ async def get_files(self):
         """Get file structure from ArangoDB in the format of
         {
             "name": "File Name",
-            "id": "File Name",
+            "id": "File Name", 
             "type": "File",
             "parent": "",
         }"""
-        pass
+        try:
+            client = ArangoClient(hosts=ARANGO_URL)
+            db = client.db(name=ARANGO_DB_NAME, username=ARANGO_USERNAME, password=ARANGO_PASSWORD, verify=True)
+        except Exception as e:
+            raise HTTPException(status_code=500, detail=f"Failed to connect to ArangoDB: {e}")
```

Comment:
I believe the Retriever class has some kind of `initialize_arangodb` helper to store this code. Can we do the same for Dataprep?